hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/health/SubscriptionHealthContextCreator.java

```diff
@@ -0,0 +1,31 @@
+package pl.allegro.tech.hermes.management.domain.subscription.health;
+
+import pl.allegro.tech.hermes.api.Subscription;
+
+import static java.lang.Double.parseDouble;
+
+class SubscriptionHealthContextCreator {
```

Comment:
Unfortunately pl.allegro.tech.hermes.api.SubscriptionMetrics conflicts with pl.allegro.tech.hermes.management.domain.subscription.health.SubscriptionMetrics (created below in convertSubscriptionMetrics()). I created the second SubscriptionMetrics class in domain because the one in API can contain arbitrary strings, e.g. "unavailable" and it would complicate SubscriptionHealthProblemIndicator implementations.